vtex-apps/store-graphql
# Add vtex_segment cookie to all catalog requests

#### What is the purpose of this pull request?
It adds vtex_segment cookie to all catalog requests

#### What problem is this solving?
Product prices are coming with no discounts related to promotions because we are not sending proper context when consuming catalog API.

#### How should this be manually tested?
This PR depends on https://github.com/vtex/render-server/pull/186 and https://github.com/vtex-apps/render-runtime/pull/126 

Also @felipebacelar will release a fix to make Portal work properly with `vtex_segment` cookie.

Then you can link all these apps and browse `https://{workspace}--lojadodandy.myvtex.com/wooden-couch/p?utm_source=descontao` to see a product with a lower price. Another test is to go to the homepage `https://{workspace}--lojadodandy.myvtex.com/?utm_source=descontao` and navigate to this product (wooden-couch).

#### Screenshots or example usage

#### Types of changes
